cx_Freeze/main.py

```diff
@@ -94,11 +94,10 @@ def ParseCommandLine():
             dest = "excludeModules",
             metavar = "NAMES",
             help = "comma separated list of modules to exclude")
-    parser.add_option("--ext-list-file",
-            dest = "extListFile",
-            metavar = "NAME",
-            help = "name of file in which to place the list of dependent files "
-                   "which were copied into the target directory")
```

Comment:
Why are you removing the --ext-list-file option?

Reply:
@anthony-tuininga  this option is unused: extListFile is not passed to Freezer, and I see no option in Freezer that could match its description, so I felt its best to remove it than silently ignore it

Reply:
@anthony-tuininga are we good ?

Reply:
Yes. I see that indeed the option is unused.